server.js

```diff
@@ -15,9 +15,9 @@ app.get('/', (req, res) => {
   res.send('smoke test');
 });
 
-app.use(home);
-app.use(about);
-app.use(contact);
+app.use('/home', home);
```

Comment:
So this isn't really needed. You could revert this change and still having all the routing properly working.